xaynetwork/xayn_discovery_app
# TB-4185 TB-4182 add topics tests and analytics

### What 🕵️ 🔍

- Add tests for topics usecases
- Track analytics events

----------

### How to test (please adjust template) 🥼 🔬
- CI indicates whether tests are passing
- Enable topics feature
- Check mixpanel when the topics card is shown

----------

### References 📝 🔗

- [JIRA](https://xainag.atlassian.net/browse/TB-4185)
- [JIRA](https://xainag.atlassian.net/browse/TB-4182)

----------
